### DISCUSSION TITLE
So I tried using 1.0.2, but I'm getting 'Class "CyclopsLevelBuilder" hides a global script class.'

### DISCUSSION THREAD

**Discussion by wmsteinhardt** [blackears/cyclopsLevelBuilder#113] [Q&A]
Actually, the first thing that happens is I get this when I try to enable the add-on:
"Unable to load addon script from path: 'res://addons/cyclops_level_builder/cyclops_level_builder.gd'. This might be due to a code error in that script.
Disabling the addon at 'res://addons/cyclops_level_builder/plugin.cfg' to prevent further errors."

To see what error pops I dropped that code into the editor, to see the "hides a global script class" error.

I also found this post:
https://godotforums.org/d/34260-sometimes-when-a-godot-project-loads-it-cannot-find-classes-or-references
so I have a feeling you'll have already figured out how to fix this.  I've tried deleting the cache - not sure what else to try.

Any insight would be very appreciated.

Edits:

If there is a way to export as .gltf with 1.01, that is my primary goal.  It's noted as an option in the documentation but the option isn't present in the menu.  

I am using 4.2 stable.  I had no trouble with 1.01, but it seems like I get these same issues if I try to add 1.02 to an empty project.  

Something else I tried, to no avail, was wiping the global_script_class_cache.cfg file of all mention of cyclops before adding, but this did not work.

Fantastic project, btw.  Love the tool!

**Comment by blackears** [comment]
The integration with the Godot editor is starting to cause issues as the plugin becomes more complex.  These are issues that only really show up at the installation stage, so I didn't really notice it when I was working on the code.  

I've released a new video.  The installation section will hopefully resolve your issue.

https://youtu.be/mbw_6dnOt_g
